cli/prettier.config.mjs

```diff
@@ -0,0 +1,15 @@
+import baseConfig from "../prettier.config.mjs";
+
+/** @type {import('prettier').Config} */
+const config = {
+  ...baseConfig,
+  arrowParens: "always",
+  plugins: [
+    await import("@ianvs/prettier-plugin-sort-imports"),
```

Comment:
https://github.com/t3-oss/create-t3-app/pull/1500#issuecomment-1634749628
this fixes this. im none the wiser

Reply:
not convinced the upgrade project is getting its imports sorted lol